### PR TITLE
Excused Absence deletion implementation

### PR DESCRIPTION
**Pull request by off-by-1-error**
in the destroy function, I added checks to verify that if you're not an admin, you can only destroy your own excused absences. Admins can still delete any excused absence

I also documented the approve function

### CODE REVIEW

File: server/api/excused_absence/excused_absence.controller.js [hunk 1]
```diff
@@ -115,9 +122,13 @@ exports.update = (req, res) => {
...
+    
+  if(auth.hasRole('admin') || (auth.isAuthenticated === ExcusedAbsence.findById(req.params.id).user )) {
+      return ExcusedAbsence.remove({ _id: req.params.id })
+      .then((response) => {
+          return res.status(200).send(response).end()
+      }).catch(next)}
+  
+  else {
+    return res.status(401).send(response).end()
+  }
```
Comment (aeksco) [inline server/api/excused_absence/excused_absence.controller.js:133]: Ooops! You're missing a `}` here to close off the `exports.destroy = (...) => {` statements

Just add a last `}` at line `135` and push that change

**Review by aeksco — CHANGES_REQUESTED** [review]
@off-by-1-error You're *literally* off by one error - otherwise fantastic work! Address my single comment and push the change - once tests pass we'll get this merged in :+1:

File: server/api/excused_absence/excused_absence.controller.js (hunk 1 above)
Comment (off-by-1-error) [reply server/api/excused_absence/excused_absence.controller.js:133]: fixed that and pushed the changes!

File: server/api/excused_absence/index.js [hunk 2]
```diff
@@ -3,11 +3,42 @@ const controller = require('./excused_absence.controller')
...
+// PUT /api/excused_absences/:id
+router.put('/:id/approve', auth.hasRole('admin'), controller.update)
+router.put('/:id/deny', auth.hasRole('admin'), controller.update)
+
 router.delete('/:id', auth.isAuthenticated(), controller.destroy)
 
 module.exports = router
+
+
+// User
```
Comment (agundy) [inline server/api/excused_absence/index.js]: This doesn't seem to be the right place for a to-do list long term. Can you remove before merging?

Comment (aeksco) [reply server/api/excused_absence/index.js]: @agundy - yep - definitely not the place. @off-by-1-error please remove and push again - thanks!

Comment (off-by-1-error) [reply server/api/excused_absence/index.js]: oops, sorry about that. It's gone now

**Pull request merged**